doc/source/comparison_with_sql.rst

```diff
@@ -228,9 +228,9 @@ Grouping by more than one column is done by passing a list of columns to the
 JOIN
 ----
 JOINs can be performed with :meth:`~pandas.DataFrame.join` or :meth:`~pandas.merge`. By default,
-:meth:`~pandas.DataFrame.join` will join the DataFrames on their indices. Each method has
+:meth:`~pandas.DataFrame.join` will join the DataFrames on their indexes. Each method has
```

Comment:
Sorry if I missed it before but any reason we changed these? `indices` seemed preferable to me

Reply:
I saw both terms in the documentation and since the spelling marked indices as wrong I changed a lot of them that's the reason why. I can revert these changes 😃 👍